.ado/jobs/desktop.yml

```diff
@@ -48,16 +48,20 @@ parameters:
           - Name: X86Debug
             BuildConfiguration: Debug
             BuildPlatform: x86
+          - Name: X86Release
```

Comment:
Curious to see if this test passes! I'm having trouble with it on the 0.74 preview branch